.circleci/workflows.yml

```diff
@@ -1447,11 +1447,11 @@ jobs:
       - run:
           name: Install extract-cloud-results package
           command: |
-            yarn add https://cdn.cypress.io/extract-cloud-results/v1/extract-cloud-results.tgz
+            yarn add https://cdn.cypress.io/extract-cloud-results/v1/extract-cloud-results.tgz -W
       - run:
           name: Verify Accessibility Results
           command: |
-            node ./scripts/verify-accessibility-results.js
+            CYPRESS_RECORD_KEY=$MAIN_RECORD_KEY node ./scripts/verify-accessibility-results.js
```

Comment:
@emilyrohrbough How will this work with contributor PRs?

Reply:
@jennifer-shehane How did you originally expect this to wire up? Is this job going to be hit by the contributor workflow? IIRC contributor PRs aren't recorded to the Cloud

Reply:
Yah I didn't think about it before til I just saw this record key. We need to have a check like we do elsewhere like here and I guess just skip the job if it's a contributor PR. https://github.com/cypress-io/cypress/blob/develop/.circleci/workflows.yml#L647

Reply:
add the job to the contributor workflow: https://github.com/cypress-io/cypress/pull/30325/commits/c62a7741e587bd695fa3f0195344aca7add42a67